sycl/test-e2e/Matrix/joint_matrix_bfloat16_impl.hpp

```diff
@@ -76,32 +76,64 @@ void matrix_multiply(big_matrix<T1, M, N> &C, big_matrix<T2, M, K> &A,
    }).wait();
 }
 
-int main() {
+template <typename T, typename TResult, size_t TM, size_t TN, size_t TK>
+void test() {
```

Comment:
Since several combinations are tested, could you add output of TMxTNxTK so it is clear which combination failed?